util/db/dbutil.go

```diff
@@ -209,21 +212,42 @@ func (db *Accessor) Atomic(fn idemFn, extras ...interface{}) (err error) {
 	}
 
 	var tx *sql.Tx
-	ctx := context.Background()
 	var conn *sql.Conn
-	conn, err = db.Handle.Conn(ctx)
+	ctx := context.Background()
+
+	for i := 0; ; i++ {
```

Comment:
use LoggedRetry?

Reply:
I want to have a different info/warn/err message to indicate that it's a connection error. If this becomes persistent, we need to have that information.